bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ClientConfiguration.java

```diff
@@ -128,7 +128,6 @@
     protected static final String PCBC_TIMEOUT_TIMER_NUM_TICKS = "pcbcTimeoutTimerNumTicks";
     protected static final String TIMEOUT_TIMER_TICK_DURATION_MS = "timeoutTimerTickDurationMs";
     protected static final String TIMEOUT_TIMER_NUM_TICKS = "timeoutTimerNumTicks";
-    protected static final String WAIT_TIMEOUT_ON_BACKPRESSURE = "waitTimeoutOnBackpressureMs";
```

Comment:
I this it would be better to have two separate properties, one for server and another for client.
Having two properties with the same name will be confiusing and/or we have to write docs to explain this fact

Reply:
will do